components/dear/sources/order-created/order-created.mjs

```diff
@@ -0,0 +1,24 @@
+import constants from "../../common/constants.mjs";
+import common from "../common.mjs";
+
+export default {
+  ...common,
+  name: "Order Created",
```

Comment:
I'm changing the name to "New Authorized Sale Order" and about emiting orders per event I don't agree because there is a feature in the UI where you can undo the order to update it and then Authorize it again so that will emit the same Sale Order but with a different trace id in this case so that would be good enough.